### DISCUSSION TITLE
Should we make Discord for Linux-Surface

### DISCUSSION THREAD

**Discussion by NP-chaonay** [linux-surface/linux-surface#879] [Ideas]
Hi this discussion is both 2 things
- official discord for linux-surface, using with matrix
- I have make unofficial discord community server for linux-surface [here](https://discord.gg/p4uszyUbUY)

**Comment by NP-chaonay** [comment]
Noted that for admin of matrix will be added as both admin/moderator role in my unofficial server.

(copied from my unofficial discord) 
I announce that if you are qzed and Dorian Stoll (both of them are admin of linux-surface matrix).
You could mention me, so I could add you to the role Admin/Moderator.

The server improvement/to-change and plan to convert this to official will be / could be talk later at #moderator-general channel
(2FA is required at this moment to using moderator function)

**Reply by StollD** [reply]
IMHO, the difference between Matrix and Discord is not that big. And I kinda dislike the idea of using proprietary tools for opensource projects. But if people want Discord, I am not opposed to having an unofficial server around. But then we kinda get back into the territory of having useful information scattered across many platforms (like back when we still used Gitter + IRC).

**Reply by NP-chaonay** [reply]
> "And I kinda dislike the idea of using proprietary tools for opensource projects."

IMHO too, because we also doing rev eng on MS Surface and also using GitHub which I think it is  proprietary tools.

I think I have different idea rather than what you commented. 

I not mean that we should use only proprietary.

But sometime we use what it is better, or we can use both.

**Reply by NP-chaonay** [reply]
> IMHO, the difference between Matrix and Discord is not that big. And I kinda dislike the idea of using proprietary tools for opensource projects. But if people want Discord, I am not opposed to having an unofficial server around. But then we kinda get back into the territory of having useful information scattered across many platforms (like back when we still used Gitter + IRC).

yes but I understand that Matrix may better at IRC integration. So I also recommended to using matrix as main communication method

**Comment by NP-chaonay** [comment]
@StollD 

(UPDATED 31/07/22 23:30 UTC+7) : I have re-edit 1) and 4)
ok here is the reason why I make discord server

1) I using Element as matrix client on my mobile, but it has very annoying bug when after sending text, it displayed misproperly
2) I think many people having Discord but not Matrix. However I also think that many people also have IRC but however if I not wrong, it do not have thread feature.
3)IMO,  Thread function in Discord may better than in Matrix (Element at least) because of notification features (i think), the discord thread wont be notify everytime if you is not joined in the thread.

**Reply by NP-chaonay** [reply]
I forget to tell that
4) Element matrix thread is not support on mobile version
5) If I not wrong, matrix is not place for thread message
6) I think there is many advantgaes over matrix
7) so wr may have two alternative
8) According to what I said,  I think Discord better at the long reply such as what I do today in matrix.

**Reply by NP-chaonay** [reply]
9) Matrix Thread is in beta and it not works well. so Discord Thread is work best
10) for long reply and many reason, Message Tread is the should-have thing, so I think we cannot rely only matrix.

**Reply by StollD** [reply]
> 4) Element matrix is not support on mobile

For what its worth, element is available on mobile:

* https://apps.apple.com/app/vector/id1083446067
* https://play.google.com/store/apps/details?id=im.vector.app

And yeah, threads are probably an afterthought on matrix, but I personally kinda find them annoying. Idk, I like the linearity of a chat.

**Reply by NP-chaonay** [reply]
> And yeah, threads are probably an afterthought on matrix, but I personally kinda find them annoying. Idk, I like the linearity of a chat.

sorry for the inaccurate/type/misunderstanding text of 1) and 4); I have reeditted.

Yeah it also depend on ones opinion. If someone prefer IRC-alike rather than thread, they would say like you.
But in the view of who prefer thread would say that IRC is not good in many case especially when long-reply msgs are involved.

Just like me, when doing long reply in Support ch in matrix, I also afraid that I will be temporarily blocked from sending text because of (similar to but it is not) spam. lamo.

And last but not least, " but I personally kinda find them annoying." idk if you mean  thread feature on every platform or just beta feature of element matrix.

**Reply by StollD** [reply]
> Just like me, when doing long reply in Support ch in matrix, I also afraid that I will be temporarily blocked from sending text because of (similar to but it is not) spam. lamo.

Heh, dont worry about that. In Support its completely fine. In the development channel it will be annoying for the IRC users, but even there its not forbidden.

> And last but not least, " but I personally kinda find them annoying." idk if you mean thread feature on every platform or just beta feature of element matrix.

On every platform. Its just not how I learned things. Let chats be chats, and forums be forums.

**Comment by NP-chaonay** [comment]
So again if there is no official, then I understand why.
But for who want to using Discord, I have made the unofficial one, and feel free for every admin in matrix to request the admin/moderator permission from my unofficial server.

But if Linux-Surface plan to make an official, you can copy my server template or even convert my existed one.

**Comment by NP-chaonay** [comment]
PS: my server is a discord community server, not regular server. I forgot to tell

all moderator function required 2FA at this moment.

**Reply by DGMurdockIII** [reply]
Please please make this a thing
